views/review.html

```diff
@@ -1,4 +1,5 @@
 {% extends "layout.html" %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
```

Comment:
Does this belong in layout.html with other imports?

Reply:
Good spot - now moved to the layout.html page.